packages/web/src/scss/components/Header/_Header.scss

```diff
@@ -10,14 +10,17 @@
     align-items: center;
     height: 68px;
     padding-inline: theme.$header-padding-x;
-    background-color: transparent;
 }
 
 .Header--simple {
     justify-content: center;
     height: 60px;
 }
 
+.Header--transparent {
```

Comment:
I should confess this modifier is not present in the `web` README. Header looks the same with or without it… Does it matter much? 👀